src/terminal/select.c

```diff
@@ -308,6 +317,10 @@ static void guac_terminal_clipboard_append_row(guac_terminal* terminal,
 
             int codepoint = buffer_row->characters[i].value;
 
+            /* Fill empty with spaces if not at end of line */
+            if (codepoint == 0 && i < eol)
+                codepoint = 32;
```

Comment:
Should probably `#define` this as a constant.

Reply:
Done!